internal/gosourcefile/gosourcefile.go

```diff
@@ -30,10 +32,22 @@ func (f *GoSourceFile) Incubate(virus viruses.Virus) []*goinfectedfile.GoInfecte
 		panic(fmt.Errorf("failed parsing file '%s': %w", f.relativePath, err))
 	}
 
+	cfg := types.Config{
+		Importer: importer.Default(),
+	}
+
+	info := types.Info{
+		Defs:  map[*ast.Ident]types.Object{},
+		Uses:  map[*ast.Ident]types.Object{},
+		Types: map[ast.Expr]types.TypeAndValue{},
+	}
```

Comment:
Here's where I'd add the linter exception (`//nolint:exhaustruct`), re: [the previous comment](https://github.com/gtramontina/ooze/pull/5/files#r1179866425).
Explaining why we're disabling it and reference the docs, perhaps:

https://github.com/golang/go/blob/go1.20.3/src/go/types/api.go#L404-L414